packages/fxa-auth-server/lib/payments/stripe.ts

```diff
@@ -2554,7 +2613,9 @@ export class StripeHelper {
         case 'invoice.payment_failed':
         case 'invoice.updated':
         case 'invoice.deleted':
-          await this.processInvoiceEventToFirestore(event);
+          if (!event.request?.id) {
+            await this.processInvoiceEventToFirestore(event);
+          }
```

Comment:
hmm I don't think we update the invoices directly in Firestore after mutations from us?

Reply:
Hmm, I'll have to check and review.

Reply:
We do not update it each time, removing this.